tensorflow_addons/callbacks/README.md

```diff
@@ -3,12 +3,12 @@
 ## Maintainers
 | Submodule  | Maintainers  | Contact Info   |
 |:---------- |:------------- |:--------------|
-|  | | |
+|  tqdm_progress_bar | @shun-lin | shunlin@google.com |
 
 ## Contents
 | Submodule | Metric  | Reference                               |
```

Comment:
Could you edit this README to say `Callback` instead of `Metric` then you can fill it in with `TQDMProgressBar`

Reply:
got it :)